core/src/main/java/com/scalar/db/io/Key.java

```diff
@@ -300,8 +331,8 @@ public String getColumnName(int i) {
    * @param i the position of the column which this key is composed of
    * @return the BOOLEAN value of the i-th column which this key is composed of
    */
-  public boolean getBoolean(int i) {
-    return values.get(i).getAsBoolean();
+  public boolean getBooleanValue(int i) {
```

Comment:
Is this a backward-incompatible change?

Reply:
Ah sorry, I should have mentioned it. It's not a backward-incompatible change because these methods are added in #515 and they are not released yet.

Reply:
Ah, I got it. Thank you!